test_runner/src/main/kotlin/ftl/gc/android/Utils.kt

```diff
@@ -23,6 +24,16 @@ internal fun Map<String, String>.mapToDeviceFiles(): List<DeviceFile> =
         )
     }
 
+internal fun Map<String, String>.mapToDeviceObbFiles(obbnames: List<String>): List<DeviceFile> {
+    var index = -1 // intentional
+    return map { (_, gcsFilePath: String) ->
```

Comment:
how about `mapIndexed` ?
```suggestion
    return values.mapIndexed { index, gcsFilePath -> ...
```

Reply:
👍